### ISSUE TITLE
Idea: Automated switching to busy status while streaming

### ISSUE THREAD

**Issue by rampa3** [Neos-Metaverse/NeosPublic#2458]
## Is your feature request related to a problem? Please describe.
Request is not related to a proble, but an idea I got while watching Nexulan streaming official Neos stream.

## Relevant issues
None exist.

## Describe the solution you'd like
The idea I propose is to add a toggle in the game, that allows you to set Neos to auto set your status to busy when you start streaming, and revert it back to original state when you stop streaming.

## Describe alternatives you've considered
The alternative everyone uses at the moment is setting their state manually.

## Additional context
This idea was mentioned in chat during 3K follower Neos stream.

**Comment by shadowpanther** [comment]
May I suggest that this status should be a separate one? So it would show that the user is streaming in the Contacts list (reworked one, of course).

**Comment by rampa3** [comment]
> 
> 
> May I suggest that this status should be a separate one? So it would show that the user is streaming in the Contacts list (reworked one, of course).

I agree - something like "Streaming"... The thing with the busy status was mainly since the target is to disable notifications while you stream.
